public/js/controllers/posts.js

```diff
@@ -1,5 +1,5 @@
 /*global flapperNews */
-flapperNews.controller('MainCtrl', [
+flapperNews.controller('MainController', [
   '$scope',
   'posts',
   function ($scope, posts) {
```

Comment:
@agconti assuming this is a typo, but did you mean:

```
// then attach
angular.module('flapperNews')
  .controller('MainController', ['$scope, 'posts', MainController])
;
```
or is there some magick with the registration of controller names that I'm not aware of?